packages/edit-navigation/src/components/layout/use-navigation-editor.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState, useEffect } from '@wordpress/element';
+
+export default function useNavigationEditor() {
+	const menus = useSelect( ( select ) =>
+		select( 'core' ).getMenus( { per_page: -1 } )
```

Comment:
Shouldn't we want to include `[]` as the second argument of `useSelect`?

Reply:
d02bcb8c86